tests/Model/PHPConst.php

```diff
@@ -39,6 +39,9 @@ public function readObjectFromStubNode($node): static
         $this->value = $this->getConstValue($node);
         $this->collectTags($node);
         $parentNode = $node->getAttribute('parent');
+        if (property_exists($parentNode, 'attrGroups')) {
```

Comment:
Do we need `if` here? If so, why it's not needed in other places as well?

Reply:
Because constants declared globally in file don't have such property, but still they are constants
<img width="447" alt="Screenshot 2020-12-17 at 16 27 25" src="https://user-images.githubusercontent.com/37301285/102494024-1a3d8700-4085-11eb-990e-e4453772a356.png">
and comparing to constants declared in class
<img width="456" alt="Screenshot 2020-12-17 at 16 30 52" src="https://user-images.githubusercontent.com/37301285/102494121-3fca9080-4085-11eb-9074-91896489ecb9.png">